### PR TITLE
chore: remove dead code

### PR DESCRIPTION
**Pull request by zdtsw**
## Description

- the same code has been called up in the Reconcile()
https://github.com/opendatahub-io/opendatahub-operator/blame/778ca9962ed79313cb389ba2772c5fd21ebb97ec/controllers/dscinitialization/dscinitialization_controller.go#L138

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by ykaliuta** [comment]
What about createOdhNamespace() calls?

**Comment by zdtsw** [comment]
> What about createOdhNamespace() calls?

i will clean the 2nd one as well.

**Review by ykaliuta — APPROVED** [review]

File: controllers/dscinitialization/dscinitialization_controller.go [hunk 1]
```diff
@@ -144,15 +144,15 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			saved.Status.Phase = status.PhaseProgressing
 		})
 		if err != nil {
-			r.Log.Error(err, "Failed to add conditions to status of DSCInitialization resource.", "DSCInitialization Request.Name", req.Name)
+			r.Log.Error(err, "Failed to add conditions to status of DSCInitialization resource.", "DSCInitialization", req.Namespace, "Request.Name", req.Name)
```
Comment (ykaliuta) [inline controllers/dscinitialization/dscinitialization_controller.go:147]: Unrelated change, btw

Comment (zdtsw) [reply controllers/dscinitialization/dscinitialization_controller.go:147]: yeah, when removed the duplicated one, saw it has better logging than the kept one, so i just use it instead
